addons/web/static/src/legacy/scss/web_calendar.scss

```diff
@@ -809,12 +820,13 @@ $o-cw-filter-avatar-size: 20px;
 }
 
 // Generated with http://phrogz.net/css/distinct-colors.html
-$o-filter_colors: #ff1e00, #bf9600, #00ff84, #00aeff, #0003b3, #ff006a, #cc3300, #ffea00, #00cc85, #007ab3,
-                    #6900cc, #ff0048, #ff6200, #c2cc00, #00ffc8, #008cff, #a600ff, #bf0036, #b24400, #d0ff00,
-                    #00ffea, #0062b3, #ea00ff, #ff0026, #ff8400, #8bcc00, #00bfaf, #006aff, #af00bf, #bf001d,
-                    #bf6300, #8cff00, #00f2ff, #004ab3, #ff00d0, #ffa600, #3acc00, #00b6bf, #0048ff, #d90094,
-                    #bf7c00, #04ff00, #00d0ff, #0036bf, #ff008c, #ffc800, #00bf49, #0092b3, #0004ff, #b20062,
-                    #649173 !default;
+// Do not modify the first 12 colors of this list or it will break the colors in the Time Off Calendar
```

Comment:
Doesn't seem to be very robust.

Why 12 colors, and not 13 ? Why do you need to hack the colors like this ? "Otherwise it will break is not sufficient".